taxonium_web_client/src/components/SearchPanel.jsx

```diff
@@ -162,6 +164,30 @@ function SearchPanel({
             </Select>
           </label>
         )}
+        <label className="space-x-2 text-sm block">
+            <span className="text-gray-500 text-sm">Treenome Browser:</span>
```

Comment:
I think if we go with `Treenome browser` it would be good to have a `title` that explains what this does.

Reply:
We could just call it Genome Browser in the app to avoid confusion -- though I think we will end up using the name Treenome Browser in publication, so might be good to match. Let me know what you think. 

If we keep the name, maybe `Treenome Browser (?)` with the ? being either a link to a section in the docs or a tooltip with a brief description?

Reply:
Yeah that sounds great to me! I think tooltip would be great, but we can do custom tooltip that includes a link to docs.

Reply:
Nice!

Reply:
Can link to https://docs.taxonium.org/en/latest/treenome.html and add a placeholder with even a couple of sentences  in `/docs/treenome.md` if you are happy to use the Taxonium docs. (Using those docs would be my minor preference, but if you prefer an external site that's fine by me).

Reply:
@amkram Pls can we try to do this before merging

Reply:
Added some docs to `treenome.md`

Reply:
I also added a short tutorial video which might be useful, as I think navigating treenome browser is slightly less intuitive than base Taxonium

Reply:
(Awesome stuff! - I was thinking I need videos too even before this and now even more convinced)